Jenkinsfile

```diff
@@ -13,6 +13,14 @@ pipeline {
   }
 
   stages {
+    stage('Validate') {
+      parallel {
```

Comment:
Why are you using parallel for just a single step? Are you planning for other specific validate steps in the future, or just planning for the potential for more validation (like with the static analysis stuff I'm working on?)

Reply:
Fixed